src/main/java/com/mael/payroll/transactions/AddMonthlyEmployee.java

```diff
@@ -0,0 +1,37 @@
+package com.mael.payroll.transactions;
+
+import com.mael.payroll.Employee;
+import com.mael.payroll.HoldMethod;
+import com.mael.payroll.PayrollDBFacade;
+import com.mael.payroll.paymentSchedules.MonthlySchedule;
+import com.mael.payroll.paymentTypes.MonthlyPayment;
+
+public class AddMonthlyEmployee implements AddEmployeeTransaction {
+
+    private int employeeId;
+    private String name;
+    private String address;
+    private double monthlyRate;
+    private PayrollDBFacade payrollDB;
+
+    public AddMonthlyEmployee(int employeeId, String name, String address, double monthlyRate) {
+        this.employeeId  = employeeId;
+        this.name        = name;
+        this.address     = address;
+        this.monthlyRate = monthlyRate;
+    }
+
+    public void execute() {
```

Comment:
To pick up the discussion from yesterday, maybe it does make sense to have an abstract class for the `AddEmployeeTransaction`.
This way you could centralise the behaviour in `execute()` - as all implementation look almost identical right now. Create an employee and add it to the payroll database.